fili-sql/src/main/java/com/yahoo/bard/webservice/sql/DruidQueryToSqlConverter.java

```diff
@@ -929,6 +939,55 @@ private void sqlAggregationHelper(
         return allGroupBys;
     }
 
+    /**
+     * Find all word positions in string.
+     * @param textString a string to search into.
+     * @param word a target word.
+     * @return an array of indexes of all positions word was found in the textString.
+     */
+    private List<Integer> findWord(String textString, String word) {
+        List<Integer> indexes = new ArrayList<>();
+        String lowerCaseTextString = textString.toLowerCase(Locale.ENGLISH);
+        String lowerCaseWord = word.toLowerCase(Locale.ENGLISH);
+        int wordLength = 0;
+
+        int index = 0;
+        while (index != -1) {
+            index = lowerCaseTextString.indexOf(lowerCaseWord, index + wordLength);
+            if (index != -1) {
+                indexes.add(index);
+            }
+            wordLength = word.length();
+        }
+        return indexes;
+    }
+
+    /**
+     * Removes the quotes around the Sketch metric literal.
+     * @param sqlQuery the sql string built by the RelBuilder.
+     * @return the sql string without quote around the sketch metric expression.
+     */
```

Comment:
Ditto on the style feedback.
Vertical whitepsace between description and params and between params and returns. Two spaces after the variable name on parameter comments.